constructr-coordination/src/main/scala/de/heikoseeberger/constructr/coordination/ConsulCoordination.scala

```diff
@@ -95,9 +129,9 @@ final class ConsulCoordination(prefix: String, clusterName: String, host: String
 
   override def initialBackendContext = ""
 
-  private def putKeyWithSession(keyUri: Uri, sessionId: SessionId)(implicit ec: ExecutionContext, mat: Materializer): Future[Boolean] = {
+  private def putKeyWithSession(keyUri: Uri, content: Strict, sessionId: SessionId)(implicit ec: ExecutionContext, mat: Materializer): Future[Boolean] = {
```

Comment:
Two things:
1. Code style: if you have to use `Strict`, please use it qualified `HttpEntity.Strict` (like `HttpEntity.Empty` above) and remove the respective import above
2. Is it really necessary to use a strict entity? Please try `HttpEntity` or `RequetEntity`.